src/pages/console/data/data-model.mdx

```diff
@@ -60,4 +60,20 @@ Applications that use an Amplify GraphQL API have limited feature availability i
 - [Binding UI Components to data](../uibuilder/databinding/), and [generating Collections](../uibuilder/collections/) is not supported
 - [Binding Forms to data, and auto-generated forms](../formbuilder/overview/) are not supported
 
-To get access to these features, turn on DataStore and select a Conflict Resolution strategy.
\ No newline at end of file
+To get access to these features, turn on DataStore and select a Conflict Resolution strategy.
+
+### Unsupported directives and field-level parameters
```

Comment:
I think it'd be nice to explain here what it means that these directives are unsupported. It means they can't use Studio's Content tab or Schema designer, but they can continue using Amplify CLI to update their schemas and they can continue to use Studio for other operations.